db/migrate/20200930185510_add_sources_to_versions.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class AddSourcesToVersions < ActiveRecord::Migration[5.2]
+  def up
+    add_column :versions, :sources, :jsonb
```

Comment:
Were we thinking of maybe "source_repos" or "repository_sources" ?

Reply:
it's sort of unfortunate that packages have the repo for the package's source code AND the package manager repo the package was downloaded from, I'm sure that will confuse the f out of somebody sometime. Is there any way to have a clear naming convention to distinguish?

Reply:
@havocp At least this is in the version model? Can you think of a better column name?

Reply:
Not really. I guess npm calls it a "registry" but nobody else does. Was hoping y'all had some ideas